<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObanalResultsWidget</class>
 <widget class="QWidget" name="ObanalResultsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ObanalResults</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Отображение результатов</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Результаты за срок:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QWidget" name="timesWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QDateTimeEdit" name="startDateTimeEdit">
             <property name="displayFormat">
              <string>dd.MM.yyyy</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>−</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="endDateTimeEdit">
             <property name="displayFormat">
              <string>dd.MM.yyyy</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updateButton">
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="dataTableWidget">
     <column>
      <property name="text">
       <string>Дата</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Величина</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Срок, ч</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Модель</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Уровень</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Центр</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Тип сетки,°</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Тип уровня</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Количество данных</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Количество записей:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="countLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
